### PR TITLE
easier installation for windows

### PR DESCRIPTION
**Pull request by tzukpolinsky**
Hi,

I added some script for full g20 dep installation and modified an existing one to accept params from command arguments in script folder

and some easy fix for cmake module for different syntax in windows than linux for mentis.

I can make Windows install doc if you would like I just didn't really saw where it will fit in the project and I didn't want to ruin anything.

Hope thats helps:)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 \.vs
 \.vscode
 \.cache
+\.idea
```
Comment (RainerKuemmerle) [inline .gitignore:7]: Seems to be unrelated

File: script/install-deps-windows.bat [hunk 2]
```diff
@@ -1,5 +1,6 @@
 @echo off
-
+set VCPKG_ROOT_DIR=%1
```
Comment (RainerKuemmerle) [inline script/install-deps-windows.bat:2]: What happens if not specified on the call to the script? We overwrite variables defined on the system level to empty?

**Comment by RainerKuemmerle** [comment]
Thank you. As I am not an active user of Windows, improving support for Windows is appreciated.
I left a couple of comments regarding the PR.

https://github.com/RainerKuemmerle/g2o#windows
We could here change the README to document what can be passed to the bat file.

File: script/install-deps-windows.bat (hunk 2 above)
Comment (tzukpolinsky) [reply script/install-deps-windows.bat:2]: yes, later in the script 
`if defined VCPKG_ROOT_DIR if /i not "%VCPKG_ROOT_DIR%"=="" set "VcPkgDir=%VCPKG_ROOT_DIR%"`

if not defined we continue with the default

**Comment by tzukpolinsky** [comment]
Hi
so I believe that I answered your queries:)

I added some lines to the readme and updated some lines as well

Hopes it helps:)

**Comment by RainerKuemmerle** [comment]
I merged manually the changes. Thank you.
